docs/examples/websocket.c

```diff
@@ -89,13 +89,12 @@ static CURLcode recv_pong(CURL *curl, const char *expected_payload)
               buffer);
     }
     else if(meta->flags & CURLWS_BINARY) {
-      fprintf(stderr, "ws: received BINARY frame of %u bytes\n",
-              (unsigned int)rlen);
+      fprintf(stderr, "ws: received BINARY frame of %u bytes\n", (int)rlen);
     }
     else {
       /* some other frame arrived. */
-      fprintf(stderr, "ws: received frame of %u bytes rflags %x\n",
-              (unsigned int)rlen, meta->flags);
+      fprintf(stderr, "ws: received frame of %u bytes rflags %x\n", (int)rlen,
```

Comment:
These two don't look right, does it? %u surely should be unsigned int like we just changed these two in 8515424df48da5?